lib/diagnostic.js

```diff
@@ -8,15 +8,25 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+let result = arguments[0];
+for (let i = 1; i < arguments.length; i++) {
+if (arguments[i] < result){
+result = arguments [i];
+}
+}
+return result;
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 const sum = function sum() {
-
+let result = 0;
+  for (let i = 0; i < arguments.length; i++){
+result += arguments;
```

Comment:
This should be ```arguments[i]``` since you're looping through ```arguments``` and you want to add the element at index ```i```.